### ISSUE TITLE
Unable to prebuild libraries with module_name listed in podspec

### ISSUE THREAD

**Issue by ryo-otake-sonix** [Subito-it/PodBuilder#61]
## Issue
We have confirmed that libraries with module_name listed in podspec are not prebuilt anymore. (In PodBuilder 4.2.0 and 4.3.0)
In this case, I have confirmed the libraries lottie-ios, ReachabilitySwift.

--- Log ---
<img src="https://user-images.githubusercontent.com/90164767/158292089-fc1e986f-8925-4dee-8aef-e8307ab9ddab.png">

[In New Copy Phase](https://github.com/Subito-it/PodBuilder/blob/master/lib/pod_builder/install.rb#L407), it seemed to be caused by the fact that [the module_name directory was being detected whether it exists or not.](https://github.com/Subito-it/PodBuilder/blob/master/lib/pod_builder/install.rb#L409-L414)

The directory actually generated was the directory of root_name, and the directory of module_name did not exist.
```
lottie-ios:  root_name -> lottie-ios,  module_name -> Lottie

ReachabilitySwift:  root_name -> ReachabilitySwift,  module_name -> Reachability
```

Therefore, it seems that 'Prebuilt items for xxx not found' was displayed and the later process was not done.

Libraries whose module_name was not listed in the podspec were prebuilt with no problems because they were detected by root_name.

## What I have tried
I tried to prebuild by overriding the module_name value to the root_name value in PodBuilder.json, but the build failed during command execution and did not work.

--- PodBuilder.json ---
<img src="https://user-images.githubusercontent.com/90164767/158296348-3521c656-4563-4ce9-9a6c-821390834dfd.png">
--- Error ---
<img src="https://user-images.githubusercontent.com/90164767/158295854-d8d8533d-a950-4e6a-840c-c45d3b76d18e.png">

## Fixed points
As a result, I was able to confirm that it is possible to prebuild [by passing root_name instead of module_name when generating the source_path.](https://github.com/Subito-it/PodBuilder/blob/master/lib/pod_builder/install.rb#L409)

--- Install.rb ---
<img src="https://user-images.githubusercontent.com/90164767/158297138-3eecd02b-bd7a-4591-a54a-bc145952e362.png">
--- Log ---
<img src="https://user-images.githubusercontent.com/90164767/158296495-d4528368-2e06-4bee-9b19-17060626a869.png">

**Comment by ryo-otake-sonix** [comment]
@tcamin 
I tried to create a PR for this issue, but could not push the changes.

--- Log ---
<img src="https://user-images.githubusercontent.com/90164767/158299864-4ce53783-8a3b-487b-a744-fd9bc505fbd5.png">


Therefore, could you please check this issue?

**Comment by tcamin** [comment]
I released version 4.4.0 which I hope should fix the issue, let me know.

> @tcamin I tried to create a PR for this issue, but could not push the changes.

FYI pull request should be opened from a fork, see https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork
